### ISSUE TITLE
FR: shortcut for dislike button

### ISSUE THREAD

**Issue by dumprop** [tiliado/nuvola-app-yandex-music#22]
Currently yandex music player have like and dislike button 
kde connect works well
![image](https://user-images.githubusercontent.com/34004367/142073228-e4aa12f9-829f-46d9-82f7-d2c2dbebfd80.png)
![image](https://user-images.githubusercontent.com/34004367/142073333-15000ad3-8d8e-4c14-841e-d9f8ae22ed24.png)

**Comment by jiri-janousek** [comment]
Hello @dumprop, thanks for your suggestion. Do the pictures show KDE desktop notifications?

@d-r-q, is it possible to integrate the like/dislike buttons? Probably as [custom actions](https://tiliado.github.io/nuvolaplayer/development/apps/mediaplayer.html#custom-actions)?

**Comment by dumprop** [comment]
> Hello @dumprop, thanks for your suggestion. Do the pictures show KDE desktop notifications?

Yes, its kde desktop notifications from kde connect

**Comment by jiri-janousek** [comment]
Nuvola runtime currently does not provide a way for Yandex Music script to add the extra actions to notifications. I've created tiliado/nuvolaplayer#774 for that.

**Comment by dumprop** [comment]
> Nuvola runtime currently does not provide a way for Yandex Music script to add the extra actions to notifications. I've created [tiliado/nuvolaplayer#774](https://github.com/tiliado/nuvolaplayer/issues/774) for that.

dislike under current like would be acceptable too :)
![image](https://user-images.githubusercontent.com/34004367/142288026-e58941df-898b-445c-bfc7-d77d29349e81.png)
but notification actions maybe helpful too

**Comment by d-r-q** [comment]
> @d-r-q, is it possible to integrate the like/dislike buttons? Probably as custom actions?

No, currently I have almost no spare time and ready only to maintain operability of current functionality.

**Comment by jiri-janousek** [comment]
Thant's reasonable, thanks for that :+1:

**Comment by jiri-janousek** [comment]
We will keep the issue in the backlog for now.
